Starls.Assets/Starls.Assets.DTO/Charactere.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Starls.Assets.DTO
+{
+    public class Charactere
```

Comment:
Corrigir typo: Character

```suggestion
    public class Character
```